app/src/main/java/org/jellyfin/androidtv/util/KeyProcessor.java

```diff
@@ -254,8 +253,9 @@ public static PopupMenu createItemMenu(BaseRowItem rowItem, UserItemDataDto user
                         && userData.getUnplayedItemCount() > 0) {
                     menu.getMenu().add(0, MENU_PLAY_FIRST_UNWATCHED, order++, R.string.lbl_play_first_unwatched);
                 }
+                boolean isSeries = item.getBaseItemType() == BaseItemType.Series || item.getBaseItemType() == BaseItemType.Season;
                 menu.getMenu().add(0, MENU_PLAY, order++, item.getIsFolderItem() ? R.string.lbl_play_all : R.string.lbl_play);
-                if (item.getIsFolderItem()) {
+                if (!isSeries && item.getIsFolderItem()) {
```

Comment:
Why disable shuffle for series? I know users have asked for this in the past.

Reply:
Jellyfin web has a shuffle button for series and seasons. I think maintaining feature parity is important.

It is nice to be able to shuffle episodes when rewatching non-linear series.

Reply:
ok, makes sense

Reply:
done